commonknowledge/join
# CK-3811 Core Support for Stripe in Minimalist Join Flow

Adds a end to end solution for using Stripe in the minimalist Join Flow.
- When adding a plan and Stripe is enabled, adds plans as products to Stripe as needed.
- Updates certain details in Stripe when it is enabled.
- Pushes the ID of the plan through to the frontend, to be used appropriately.

## Motivation and Context




See https://linear.app/commonknowledge/issue/CK-3811/core-support-for-stripe-in-minimalist-join-form

## How Can It Be Tested?




1. Set up Stripe settings in Join Flow backend.
2. Add Minimalist Join Flow to a page.
3. Observe it works end to end.

## How Will This Be Deployed?



Usual processes.

## Types of changes

- Bug fix (non-breaking change which fixes an issue)
- Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:


- My change requires a change to the documentation.
- I've updated the documentation accordingly.
